### PR TITLE
Assorted ACL corrections, mostly OCR errors

### PR DESCRIPTION
**Pull request by davidweichiang**
Up to 1999

### CODE REVIEW

**Comment by mjpost** [comment]
How did you do this?

**Comment by davidweichiang** [comment]
Just by looking at the index pages for these conferences in the Anthology website -- nothing automatic.

File: data/xml/P84.xml [hunk 1]
```diff
@@ -455,23 +455,22 @@
     </paper>
     <paper id="60">
       <title>Linguistically Motivated Descriptive Term Selection</title>
-      <author><first>K. Sparck</first><last>Jones</last></author>
+      <author><first>K.</first><last>Sparck Jones</last></author>
       <author><first>J.I.</first><last>Tait</last></author>
```
[the remark this answers is not in the document]
Comment (davidweichiang) [reply data/xml/P84.xml]: In this particular paper, there's no umlaut.

**Comment by davidweichiang** [comment]
I fixed all instances of Karen Sparck Jones, but didn't change all instances of Keith Vander Linden -- I figure at some point someone should go through and fix all the "van"s.

**Comment by davidweichiang** [comment]
Changed my mind and fixed Vander Linden, since you're sure it's right and I'm not sure about the other vans.

**Pull request merged**